packages/fxa-content-server/app/tests/spec/views/mixins/password-mixin.js

```diff
@@ -126,17 +126,6 @@ describe('views/mixins/password-mixin', function() {
         assert.equal(view.$('#password').attr('type'), 'password');
       });
 
-      it('works with touch events', () => {
```

Comment:
Mouse events are simulated in mobile devices but `touchstart` is fired first: https://developer.mozilla.org/en-US/docs/Web/API/Touch_events

This was causing the toggle to fire twice, I'm only see the password reveal for about 1/2 a second on prod. I could have also called `stopPropogation` if it's a `touchstart` event, that just feels unnecessary. `touchstart` should be used if there's functionality (like swiping) that can't be done in desktop.